src/main/java/org/opentripplanner/middleware/models/TripMonitorNotification.java

```diff
@@ -13,6 +14,7 @@
  */
 public class TripMonitorNotification extends Model {
     private static final Logger LOG = LoggerFactory.getLogger(TripMonitorNotification.class);
+    public static final String STOPWATCH_ICON = "⏱";
```

Comment:
Assuming the strings aren't unicode?

Reply:
Which portion would be not-Unicode?

Reply:
I guess I'm curious why we don't put the stop watch icon into the localized string?

Reply:
I guess it was a case of not repeating the icon, as in the trip alert notifications.

Reply:
Not a blocker. Happy to merge